File: 2139.py
while True:
  try:
    mes, dia = map(int,input().split(" "))
    soma = 0
    meses = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30]
    if mes == 12:
      if dia > 25:
        print('Ja passou!')
      else:
        print('E natal!') if dia == 25 else print('E vespera de natal!')
    else:
      for i in range(mes, 11):
        soma += meses[i]
      print(f'Faltam {soma+25+(meses[mes-1]-dia)} dias para o natal!')
  except EOFError:
    break

File: 2139.cpp
#include <iostream>
using namespace std;
int main(){
    int meses[] = {31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30};
    int mes, dia, soma;
    while(cin >> mes){
        cin >> dia;
        soma = 0;
        if(mes == 12){
            if(dia > 25)
                cout << "Ja passou!\n";
            else{
                if(dia == 25)
                    cout << "E natal!\n";
                else
                    cout << "E vespera de natal!\n";
            }
        }
        else{
            for(int i = mes; i<11; i++)
                soma += meses[i];
            cout << "Faltam " << soma+25+(meses[mes-1]-dia) << " dias para o natal!\n";
        }
    }
}
